x-pack/plugins/event_log/server/es/context.ts

```diff
@@ -70,13 +70,17 @@ class EsContextImpl implements EsContext {
         this.logger.debug(`readySignal.signal(${success})`);
         this.readySignal.signal(success);
       } catch (err) {
-        this.logger.debug('readySignal.signal(false)');
+        this.logger.debug(`readySignal.signal(false), reason: ${err.message}`);
         this.readySignal.signal(false);
       }
     });
   }
 
   async shutdown() {
+    if (!this.readySignal.isEmitted()) {
+      this.logger.debug('readySignal.signal(false); reason: Kibana server is shutting down');
+      this.readySignal.signal(false);
```

Comment:
This is the root change of this PR. Here we emit an "initialization failed" signal to [unblock](https://github.com/elastic/kibana/blob/main/x-pack/plugins/event_log/server/es/cluster_client_adapter.ts#L103) `ClusterClientAdapter`.